BaseStation/PTZCamera/PTZCamera.cs

```diff
@@ -10,14 +10,23 @@
 
 namespace HuskyRobotics
 {
-
+    /// <summary>
+    /// This class represents the IP camera and allows for connection via IP and control using HTTP GET requests.
+    /// </summary>
     class PTZCamera
     {
-        private string _base_url { get; set; }
-        private string base64_Credentials { get; set; }
-        private string _headers { get; set; }
-        public HttpClient client { get; }
+        private string _base_url { get; set; } //the ip address for the camera
+        private string base64_Credentials { get; set; } // The username and password for the camera
+        private string _headers { get; set; } //The HTTP headers for the camera
+        public HttpClient client { get; set; } // The Httpclient used to connet to and communicate with ip devices using HTTP requests. This should be initialized elsewhere and share between all HTTP based devices.
```

Comment:
We probably don't want a user to be able to set this. Since it's passed into the constructor, leave this as a get and have a private HttpClient field that the get just returns. Also make it called Client to stick with naming convention.